packages/@azure/servicebus/data-plane/lib/core/messageReceiver.ts

```diff
@@ -119,12 +112,14 @@ export class MessageReceiver extends LinkEntity {
    */
   receiverType: ReceiverType;
   /**
-   * @property {number} [maxConcurrentCalls] The maximum number of messages that should be
-   * processed concurrently while in peek lock mode. Once this limit has been reached, more
-   * messages will not be received until messages currently being processed have been settled.
-   * Default: 1
+   * @property {number} [maxConcurrentMessages] The maximum number of messages that can be
+   * fetched over the network at a time when .
+   * For more information, refer to the underlying messaging library - https://github.com/amqp/rhea.
+   * - **Default**: `1000`
```

Comment:
Default for `maxConcurrentMessages` is actually 1.

Default of 1000 is for `credit_window`